README.md

```diff
@@ -1 +1,78 @@
-# seroai-trusthub
\ No newline at end of file
+# Seroai Trusthub
+
+<!--About the Seroai Trusthub-->
+
+<p align="center">
+   <a href="https://github.com/seroai-coders/seroai-trusthub/stargazers"><img src="https://img.shields.io/github/stars/seroai-coders/seroai-trusthub" alt="Github Stars"></a>
+   <a href="https://github.com/seroai-coders/seroai-trusthub/issues"><img src="https://img.shields.io/github/issues/seroai-coders/seroai-trusthub" alt="Github issues"></a>
+    <a href="https://github.com/seroai-coders/seroai-trusthub/blob/main/LICENSE"><img src="https://img.shields.io/github/license/seroai-coders/seroai-trusthub" alt="license"></a>
+</p>
+
+## 🛠️ Built With
+
+- [Next.js](https://nextjs.org)
+- [Tailwind CSS](https://tailwindcss.com)
+- [Postgres](https://www.postgresql.org)
+- [React](https://reactjs.org)
+- [Prisma](https://www.prisma.io)
+- [TypeScript](https://www.typescriptlang.org)
+
+## ✨ Getting Started
+
+Please follow these simple steps to get a local copy up and running.
+
+### Prerequisites
+
+- Node.js (Version: >=18.x)

```

Comment:
I think we have 20.x . Please confirm. Thanks!

Reply:
@trustCoder101 Changed it to v21.x.